src/main/webapp/src/app/schedule-page/datepicker/datepicker.component.ts

```diff
@@ -42,7 +42,7 @@ export class DatepickerComponent implements OnInit {
 
   /**
    * Formats a string containing a date to: {Day Abbreviation}: {Month} {Day}, {Year}
-   * @param {string} date - a string separated by spaces with the following info {Day} {} 
+   * @param {string} date - a string separated by spaces with the following info {Day} {Month} {Day} {Year} {Hour:Minute:Seconds} {GMT} {Timezone}
```

Comment:
nitty nit: Do we need the `{string}` annotation for Typescript?

Reply:
I added it because of how the string is formatted

Reply:
The `{}` around day, month, and year are completely fine. I was just talking about `@param {string} date`. I'm just suggesting if we structure our doc comments according to http://typedoc.org/guides/doccomments/, then we can use `@param date` instead.

Reply:
Got it! I'll make the quick fix